### PR TITLE
Elements Panel: Update 'Text Set' buttons

### LINKED ISSUE #6214: Redesign: Update text set buttons
## Feature Description

Update design for text sets based on [Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=257%3A817).

- [ ] Update normal, hover and focus styling for individual text sets ([additional Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3360%3A63235))
- [ ] Update position of font match and headline
- [ ] Update toggle from design system
- [ ] Update tooltip to new component where applicable

| Current design | New design |
|-|-|
| Screen Shot 2021-02-05 at 00 06 53 | Screen Shot 2021-02-05 at 00 06 19 | 

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by samwhale**
## Context

Most of this ticket has already been completed here: https://github.com/google/web-stories-wp/pull/6549/files#diff-3e19b72f426437c96c76f87b841cac02e2c993643af2f06db00eb77b7709147aR38-R60. This PR finishes the styling for this panel.

Update design for text sets based on [Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=257%3A817).

- Update normal, hover and focus styling for individual text sets ([additional Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3360%3A63235))

**Designs**
|||
|--|--|
|image|image|

## Summary

Update remaining text set button styling

## Relevant Technical Choices

n/a

## To-do

none

## User-facing changes

|Before|After|
|--|--|
|before|text-sets|

## Testing Instructions

1. Open editor
2. Go to elements panel and select the 'text' button
3. Play with text sets

### QA

- [ ] This is a non-user-facing change and requires no QA

1. Open editor
2. Go to elements panel and select the 'text' button
3. Play with text sets

### UAT

- [x] UAT should use the same steps as above.

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6214

### CODE REVIEW

File: assets/src/edit-story/components/library/panes/text/textSets/textSet.js [hunk 1]
```diff
@@ -49,9 +49,12 @@ const TextSetItem = styled.div`
   background-color: ${({ theme }) =>
     theme.colors.interactiveBg.secondaryNormal};
   border-radius: ${({ theme }) => theme.borders.radius.small};
-  cursor: default;
+  cursor: pointer;
+  transition: background-color 0.3s;
```
Comment (BrittanyIRL) [inline assets/src/edit-story/components/library/panes/text/textSets/textSet.js]: if ya want, `BUTTON_TRANSITION_TIMING` is available for import and would align this transition with other buttons, even though these don't look the same.

**Review by BrittanyIRL — APPROVED** [review]
LGTM! 

Will the filter pills be updates separately? 

I think we should add in a message when there are no text sets matching - thoughts on a new ticket for this? 
Screen Shot 2021-03-16 at 10 53 58 AM

**Comment by miina** [comment]
I also had the same question about the pills group and the chevron button to expand-collapse. It's not in the description of the original issue, however, there's no separate issue either.

@barklund -- there's no issue for the pills group of the text sets currently. Assuming it would ideally be part of the current sprint -- were you considering as part of this issue? Or is that for a later sprint? Thoughts?

**Comment by samwhale** [comment]
> I also had the same question about the pills group and the chevron button to expand-collapse. It's not in the description of the original issue, however, there's no separate issue either.
> 
> @barklund -- there's no issue for the pills group of the text sets currently. Assuming it would ideally be part of the current sprint -- were you considering as part of this issue? Or is that for a later sprint? Thoughts?

I could do that real quick in this ticket too if y'all want, it's not too much more work but if we don't want scope creep I'm happy to make a different ticket

**Comment by samwhale** [comment]
> Will the filter pills be updates separately?

Let's see what Morten says 😄 

> I think we should add in a message when there are no text sets matching - thoughts on a new ticket for this?

Totally - ~I'll add one~ #6806 ✅

**Review by maxyinger — APPROVED** [review]

**Review by embarks — APPROVED** [review]
looks great!

**Review by barklund — APPROVED** [review]
Looks good. Let's create a new ticket for updating the pills to the pill group component from the design system - the same thing should be done in the external media tab (twice even).

**Comment by samwhale** [comment]
> Looks good. Let's create a new ticket for updating the pills to the pill group component from the design system - the same thing should be done in the external media tab (twice even).

Just made those issues here: #6873 and #6874

**Pull request merged**